editor/components/url-input/index.js

```diff
@@ -239,4 +240,17 @@ class UrlInput extends Component {
 	}
 }
 
-export default withSpokenMessages( withInstanceId( UrlInput ) );
+export class UrlInput extends URLInput {
+	constructor() {
+		super( ...arguments );
+
+		deprecated( 'wp.editor.UrlInput', {
+			alternative: 'wp.editor.URLInput',
+			plugin: 'Gutenberg',
+			version: 'v3.4',
```

Comment:
`3.5` again, I'm not quite sure what version should be used in general, but we already have a few entries scheduled for `3.5`: https://github.com/WordPress/gutenberg/blob/master/docs/reference/deprecated.md#350.

We should also add this one to the list.